documentation/modules/proc-installing-using-bundle.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-openshift.adoc
+
+[id='installing-using-bundle-{context}']
+= Installing {ProductName} using a YAML bundle
+
+The simplest way to install {ProductName} is to use the predefined YAML bundles. 
+
+.Prerequisites
+
+* (Optional) Replace namespace if you want to deploy to something other than `{ProductNamespace}` (and substitute `{ProductNamespace}` in subsequent steps):
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+sed -i 's/{ProductNamespace}/_<my-namespace>_/' install/bundles/enmasse-with-standard-authservice/*.yaml
+----
+
+.Procedure 
+
+ifeval::["{cmdcli}" == "oc"]
+. Login as as a user with cluster-admin 
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} login -u system:admin
+----
+
+. Create the project where you want to deploy {ProductName}
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} new-project {ProductNamespace}
+----
+endif::[]
+ifeval::["{cmdcli}" == "kubectl"]
+. Create the namespace where you want to deploy {ProductName}
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create namespace {ProductNamespace}
+{cmdcli} config set-context $(kubectl config current-context) --namespace={ProductNamespace}
+----
+
```

Comment:
Add:
"where <{ProductNamespace}> is the namespace where {ProductName} was deployed."